doc/conf.py

```diff
@@ -30,6 +30,30 @@
 lexers[OpamLexer.name] = OpamLexer()
 lexers[CramLexer.name] = CramLexer()
 
+from pathlib import Path
+import re
+
+# Read constant from a file
+version_file = Path(__file__).parent.parent / "otherlibs/dune-rpc/private/types.ml"
+text = version_file.read_text()
+
+match = re.search(r"\s*let\s+latest\s*=\s*(\d+),\s*(\d+)", text)
+if not match:
```

Comment:
Nitpicky: what do you think about adding also a check on the size of the group? It allows failing with a more understandable error message.

Reply:
Sure, but what is the size of the group here and how do I find it?

Reply:
I have looked at the [documentation](https://docs.python.org/3/library/re.html#re.Match.groups()), and it seems it returns a tuple anyway. The solution seems to check `major` and `minor` are not `None`:
```python
if major != None and minor != None:
  raise RuntimeError("Could not extract major and minor version")
```